packages/cuid/cuid.0.2/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer:   "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+authors:      "Marco Aurélio da Silva <marcoonroad@gmail.com>"
+bug-reports:  "https://github.com/marcoonroad/ocaml-cuid/issues"
+license:      "MIT"
+homepage:     "https://github.com/marcoonroad/ocaml-cuid"
+dev-repo:     "git://github.com/marcoonroad/ocaml-cuid.git"
+synopsis:     "CUID generator for OCaml"
+
+build: [
+    [ "dune" "subst" ] {pinned}
+    [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+run-test: [ "dune" "runtest" "-p" name "-j" jobs]
+
+depends: [
+    "ocaml" {>= "4.03.0"}
+    "alcotest" {with-test}
+    "re"       {with-test}
+    "dune" {>= "1.0"}
+    "core"
```

Comment:
just an unrelated comment (good to merge once the CI is green): have you tried to use `base` instead of `core`? It seems to be a fairly large dependency for just one module.

Reply:
Ah, thanks! What are the major differences between the two packages?
Also, it seems that my local master branch merge into release branch got some issues, for instance, duplicated jbuild and dune files. Sorry, I'll fix the things soon as possible.

Reply:
`core` is a standard library built on top of `core_kernel` which is itself a standard library built on top of `base`. Core has more things and rely on more libraries than core_kernel which itself has more things and rely on more libraries than base. However base is pretty complete in itself already so I was wondering if there was a reason core was used instead

Reply:
Nevermind, I just know `core` from the OCaml book. :relieved:
Thank you for the pro tip, my next libraries releases will aim lightweight dependencies.